providers/class.two-factor-email.php

```diff
@@ -39,8 +39,8 @@ function delete_token( $user_id ) {
 	function generate_and_email_token( $user ) {
 		$token = $this->generate_token( $user->ID );
 
-		$subject = sprintf( __( 'Your login confirmation code for %s', 'two-factor' ), get_bloginfo( 'name' ) );
-		$message = sprintf( __( 'Enter %s to log in.', 'two-factor' ), $token );
+		$subject = sprintf( esc_html__( 'Your login confirmation code for %s', 'two-factor' ), get_bloginfo( 'name' ) );
+		$message = sprintf( esc_html__( 'Enter %s to log in.', 'two-factor' ), $token );
```

Comment:
`wp_mail()` by default does not use an HTML content type, it sends as plain text.  As such, escaping it with an HTML context would be inaccurate and potentially break things.